features/project/positive/positive _accounts.feature

```diff
@@ -0,0 +1,37 @@
+@positive1
```

Comment:
But I still see an space on file name: positive  _accounts.feature, this name should be posivite_accounts.feature
                                                                   ---^ here is one space
And ... are you sure this tag is correctly? @positive1?

Reply:
Can you see the blank space between word "positive" and "_" lower dash?